src/util/address.rs

```diff
@@ -352,10 +352,27 @@ impl Address {
     pub fn script_pubkey(&self) -> script::Script {
         self.payload.script_pubkey()
     }
+
+    /// Creates a URI string *bitcoin:address* optimized to be encoded in QR codes.
+    ///
+    /// If the address is bech32, both the schema and the address become uppercase.
+    /// If the address is base58, the schema is lowercase and the address is left mixed case.
+    ///
+    /// Quoting BIP 173 "inside QR codes uppercase SHOULD be used, as those permit the use of
+    /// alphanumeric mode, which is 45% more compact than the normal byte mode."
+    pub fn to_qr_uri(&self) -> String {
```

Comment:
Still not sure about the best naming, since the function returns URI string, not URI object

Reply:
It's good enough imo, I'm no fan of unneccesarily duplicating the signature in the name and the returned string is a valid URI.